dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -565,6 +565,48 @@ class Api::V1::Pd::WorkshopsControllerTest < ::ActionController::TestCase
     assert_equal expected_workshops.map(&:id).sort, response.map {|x| x['id']}.sort
   end
 
+  test 'Teachercon workshops are public' do
```

Comment:
Augment this test to make sure that it only retrieves the teachercons this regional partner is associated with